topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -1859,6 +1859,97 @@ Now that your production-ready Galaxy is running, try registering a new user and
+    </plugins>
+    <destinations>
+        <destination id="local" runner="local"/>
+    </destinations>
+    <tools>
+    </tools>
+</job_conf>
+```
+
+The above `job_conf.xml` file defines a *plugin* and *destination* to allow Galaxy to run user jobs on the local computer (i.e. The computer that Galaxy is running on.)
+
+Firstly, the plugins section contains a plugin called "local" which is of type "runner" and then loads the python code module for supporting local jobs. Next the destinations section contains a destination called "local" using the runner "local". As this is the only destination specified, it is also the default. So now everytime a user clicks "Execute" on a tool form, Galaxy will run the corresponding job locally using the python code specified.
+
+There are a lot of other plugins available for Galaxy for using other resources such as docker containers, kubernetes hosts, Pulsar destinations and HPC clusters to name a few. See the Galaxy documentation on [job configuration](https://docs.galaxyproject.org/en/master/admin/jobs.html) for more details on these plugins and their configuration. There is also an advanced sample job conf file located at: `/srv/galaxy/server/lib/galaxy/config/sample/job_conf.xml.sample_advanced`
+
+{% endraw %}
+
+> ### {% icon hands_on %} Hands-on: Job Conf
+>
+> 1. If the folder does not exist, create `templates/galaxy/config` next to your `galaxy.yml` playbook (`mkdir -p templates/galaxy/config/`).
+>
+> 2. Create `templates/galaxy/config/job_conf.xml.j2` with the following contents:
+>
+>    ```xml
+>    <job_conf>
+>        <plugins workers="4">
+>            <plugin id="local" type="runner" load="galaxy.jobs.runners.local:LocalJobRunner"/>
+>        </plugins>
+>        <destinations default="local">
+>            <destination id="local" runner="local"/>
+>        </destinations>
+>        <tools>
+>        </tools>
+>    </job_conf>
+>    ```
+>
+> 3. Inform `galaxyproject.galaxy` of where you would like the `job_conf.xml` to reside in your group variables:
+>
+>    {% raw %}
+>    ```yaml
+>    galaxy_config:
+>      galaxy:
+>        # ... existing configuration options in the `galaxy` section ...
+>        job_config_file: "{{ galaxy_config_dir }}/job_conf.xml"
+>    ```
+>    {% endraw %}
+>
+>    And then deploy the new config file using the `galaxy_config_templates` var in your group vars:
+>
+>    {% raw %}
+>    ```yaml
+>    galaxy_config_templates:
+>      # ... possible existing config file definitions
+>      - src: templates/galaxy/config/job_conf.xml.j2
+>        dest: "{{ galaxy_config.galaxy.job_config_file }}"
+>    ```
+>    {% endraw %}
+>
+> 4. Run the playbook. At the very end, you should see output like the following indicating that Galaxy has been restarted:
+>
+>    ```
+>    RUNNING HANDLER [restart galaxy] ****************************************
+>    changed: [galaxy.example.org]
+>    ```
+> 5. Checkout the new job_conf.xml file in `/srv/galaxy/config`.
```

Comment:
could use one of the IO blocks here to show `cat /srv/galaxy/config/job_conf` and showing the output? Dunno if that's more instructive, but the green box might act as a better call to action than a bullet point.

Reply:
Good thinking. I'll add that in